packages/nameguard-sdk/src/index.ts

```diff
@@ -100,7 +103,8 @@ export enum Rating {
 export type SecurePrimaryNameStatus =
   | "normalized" /** The ENS primary name was found and it is normalized. */
   | "no_primary_name" /** The ENS primary name was not found. */
-  | "unnormalized" /** The ENS primary name was found, but it is not normalized. */;
+  | "unnormalized" /** The ENS primary name was found, but it is not normalized. */
+  | "uninspected" /** `willInspectName(name) === false` therefore the name was not inspected for performance reasons */;
```

Comment:
@lightwalker-eth `willInspectName` is hard to implement as it needs to resolve labelhashes and check length of resulted name

Reply:
@djstrong That's fair. Thanks for feedback. I suppose we shouldn't write that function then and instead we can just have an abstract definition for the concept of what would be inspected or not and refer to that idea in our comments.